### PR TITLE
#634: Scraper which scrapes a given weibo user's information

### LINKED ISSUE #634: [Weibo] Scrape user profile page and find profile user pages of followers and following

### PR DESCRIPTION
**Pull request by daminisatya**
Issue #634 : 
This PR aims at scraping the user's bio information from Weibo accounts. The following is the sample [URL](http://weibo.com/p/1005053472041784/info?mod=pedit_more) which is the bio information page. This patch needs some help with getting the source code from the above URL. 
If successful, the code can be tested properly.

### CODE REVIEW

**Comment by daminisatya** [comment]
@DengYiping @sudheesh001 Please have a look at the problem. I'm unable to get the source code from the weibo urls. Jsoup alone is not able to get the html page. Is there any other possible way for doing this? Other than adding the personal browser cookies! 

@DengYiping Also please check on how we can queue this service, and use your patch into this. #735 

Thank you.

**Comment by jig08** [comment]
@daminisatya The issue is that all the information resides inside the `<script>..</script>` tags. 
The way out is to parse the javascript code. 

As mentioned [here](http://stackoverflow.com/a/14925848), "JSoup + Manual Parsing" seems like a good and a viable option.

**Comment by DengYiping** [comment]
#735 This is the helper function implementing the idea of @jig08

**Comment by DengYiping** [comment]
@daminisatya How's it going on this PR? Anything I can help?

File: src/org/loklak/api/search/WeiboUserInfo.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	@Override
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		Query post = RemoteAccess.evaluate(request);
+
+		// manage DoS
+		if (post.isDoS_blackout()) {response.sendError(503, "your request frequency is too high"); return;}
+		
+		String url = post.get("url", "");
+		JSONObject obj = new JSONObject();
+		Document doc = Jsoup.connect(url).get();
```
Comment (DengYiping) [inline src/org/loklak/api/search/WeiboUserInfo.java]: @daminisatya Here, you should use the helper function.

**Pull request merged**